lizmap/modules/lizmap/lib/Request/OGCRequest.php

```diff
@@ -271,4 +259,27 @@ protected function getcapabilities()
             'cached' => $cached,
         );
     }
+
+    protected function logXmlError($xmldata)
```

Comment:
The name of the method does not correspond to what it does. `loadXmlString` is a better name IMHO.